applications/PfemFluidDynamicsApplication/custom_utilities/move_mesh_utility.h

```diff
@@ -34,38 +34,28 @@ namespace  Kratos
     /**
      * @class MoveMeshUtility
      *
-     * @brief This utilitiy moves the nodes of a pfem fluid
-     *        used especially in the pfem-fem coupling
+     * @brief This utility resets the nodes of a PFEM fluid
+     *        ModelPart, used especially in the PFEM-FEM coupling
      *
      * @author Klaus B Sautter
      */
     class  KRATOS_API(PFEM_FLUID_DYNAMICS_APPLICATION) MoveMeshUtility
     {
         public:
-            typedef ModelPart::NodesContainerType NodesContainerType;
+            typedef Node<3> NodeType;
 
             KRATOS_CLASS_POINTER_DEFINITION(MoveMeshUtility);
 
-            MoveMeshUtility() {};
-            ~MoveMeshUtility() {};
-
-            /**
-             * @brief This function is the main operation of this utility.
-             *        It moves the nodes of a given model part w.r.t. the current DISPLACEMENT
-             * @param rNodes The nodes array of a model part
-             */
-            void MovePfemMesh(NodesContainerType& rNodes) const;
+            MoveMeshUtility() = default;
```

Comment:
@philbucher I still have the same issue as I mentioned [here](https://github.com/KratosMultiphysics/Kratos/pull/9294#discussion_r752482809), the code doesn't compile if I delete the constructor here. I did change the binding to be static as well.